_extensions/posit-docs/theme-dark.scss

```diff
@@ -0,0 +1,395 @@
+/*-- scss:defaults --*/
+
+// import font
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap');
+
+
+/*-- scss:variables --*/
+
+// scss-docs-start color-variables
+$posit-blue: #447099;
+$posit-light-blue-1: #D1DBE5;
+$posit-light-blue-2:  #A2B8CB;
+$posit-light-blue-3: #7494B1;
+$posit-dark-blue-1: #305775;
+$posit-dark-blue-2: #213D4F;
+$posit-orange:  #EE6331;
+$posit-light-orange-1: #EDCCBF;
+$posit-light-orange-2: #EBA38C;
+$posit-dark-orange-1: #Ab4D26;
+$posit-dark-orange-2: #80361C;
+$posit-dark-orange-3: #451F12;
+$posit-gray:    #404041;
+$posit-light-gray-1: #C2C2C4;
+$posit-light-gray-2: #949494;
+$posit-dark-gray-1: #333333;
+$posit-teal: #419599;
+$posit-light-teal-1: #C2D9D9;
+$posit-light-teal-2: #94BDBF;
+$posit-light-teal-3: #70A3A6;
+$posit-dark-teal-1: #297075;
+$posit-dark-teal-2: #1F4F4F;
+$posit-dark-teal-3: #122B2B;
+$posit-green:   #72994E;
+$posit-burgundy:#9A4665;
+$posit-light-burgundy-1:#D9C4CC;
+$posit-light-burgundy-2:#BF96A3;
+$posit-light-burgundy-3: #A67380;
+$posit-dark-burgundy-1:#78384F;
+$posit-dark-burgundy-2:#542938;
+$posit-dark-burgundy-3:#2E171F;
+$primary: $posit-teal;
+$dark-md-body: $posit-light-gray-1;
+$dark-md-background: #0C1721;
+
+// Feature preview heading colors
+$preview-header: $posit-orange; /* posit orange, contrast: 8.45 */
+$preview-header-border: darken($preview-header, 5%);
+
+// scss-docs-end color-variables
+
+// Typography
+// Font, line-height, and color for body text, headings, and more.
+
+//scss-font-start font-variables
+$font-family-sans-serif: "Open Sans", "Segoe UI", Roboto, "Helvetica Neue", "Noto Sans", "Liberation Sans", Arial, sans-serif, system-ui;
+$font-family-monospace: "Source Code Pro", monospace;
+$font-family-emoji: "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+$font-size-base: 1rem;
+// scss-docs-end font-variables
+
+//font and body color
+
+$body-color: $dark-md-body;
+$popover-bg: $posit-green !default; // check this - citation previews
```

Comment:
```
// check this - citation previews
```

does something here still need to be checked?

Reply:
Verified and comment removed! Thank you!